### ISSUE TITLE
Show zero percent growth instead of infinite

### ISSUE THREAD

**Issue by danieldraper** [botmetrics/botmetrics#16]
When there is no previous data `Inf%` is displayed. Displaying 0% might be a more user friendly metric.

![screen shot 2016-09-29 at 2 30 24 pm](https://cloud.githubusercontent.com/assets/1016423/18941100/75822906-8651-11e6-93bc-f0913e3573d8.png)

Possible solution in `/app/lib/growth_calculator.rb`

```ruby
def call
  previous, current = values[position-1..position]

  return nil if current == 0

  growth = (current - previous).to_f / previous
  growth = 0.0 if growth.infinite?
  growth
end
```

**Comment by arunthampi** [comment]
@danieldraper good catch, it's a pending issue we needed to fix :)

maybe don't show anything `if growth.infinite`?

**Comment by arunthampi** [comment]
would you like to whip up a quick pull request?

**Comment by OrthoDex** [comment]
Hi!
I would like to work on this to get to know the project better :)

**Comment by OrthoDex** [comment]
Submitted a pull request at #25 . :)

**Comment by arunthampi** [comment]
Closed by #25 thanks @OrthoDex 🙌 !
